provider/pkg/provider/deployment_settings.go

```diff
@@ -577,33 +676,62 @@ func (ds *PulumiServiceDeploymentSettingsResource) Read(req *pulumirpc.ReadReque
 		DeploymentSettings: *settings,
 	}
 
+	var plaintextSettings *pulumiapi.DeploymentSettings = nil
+	var ciphertextSettings *pulumiapi.DeploymentSettings = nil
```

Comment:
convention is to just leave these uninitiated

```suggestion
	var plaintextSettings *pulumiapi.DeploymentSettings
	var ciphertextSettings *pulumiapi.DeploymentSettings
```

Reply:
Ack, will update, I just personally like express defaults